src/api/OrderAPI.ts

```diff
@@ -292,6 +283,24 @@ export class OrderAPI {
     this.assert.common.greaterThanZero(quantityToFill, coreAPIErrors.QUANTITY_NEEDS_TO_BE_POSITIVE(quantityToFill));
     this.assert.common.isNotEmptyArray(orders, coreAPIErrors.EMPTY_ARRAY('orders'));
 
+    let ordersTakerTokenAmount = SetProtocolUtils.CONSTANTS.ZERO;
```

Comment:
I started thinking about assertions too. There's a card here with some pseudocode.

I'll start writing out the 2 functions I laid out in the card below, but I wanted to give you a heads up that I'll be starting the individual order validations.

https://trello.com/c/YzVIxeun/436-validations-for-takerwallet-and-0x-orders

Reply:
yea i'm following trello too and linking the trello cards to the PRs so that we don't overlap